modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java

```diff
@@ -1336,6 +1282,15 @@ private void baselinePrint0(VisorBaselineTaskResult res) {
             ": softTimeout=" + autoAdjustSettings.softTimeout
```

Comment:
Can you please also fix my mistake? Message should look like "... enabled [softTimeout=*]", not as it is now.